content/en/real_user_monitoring/browser/troubleshooting.md

```diff
@@ -130,6 +130,10 @@ On Chromium-based browsers, when the RUM Browser SDK sends data to the Datadog i
 
 The warning is shown because the intake returns a non-empty JSON object. This behavior is a reported [Chromium issue][8]. It does not impact the RUM Browser SDK and can safely be ignored.
 
+## "Deobfuscation failed" warning
+
+A warning appears when deobfuscation fails for a stack trace. If the stack trace is not obfuscated to begin with, you can ignore this warning. Otherwise, use the [RUM Debug Symbols page][14] to view all your uploaded dSYMs. See [Investigate Obfuscated Stack Traces with RUM Debug Symbols][15].
```

Comment:
```suggestion
A warning appears when deobfuscation fails for a stack trace. If the stack trace is not obfuscated to begin with, you can ignore this warning. Otherwise, use the [RUM Debug Symbols page][14] to view all your uploaded source maps. See [Investigate Obfuscated Stack Traces with RUM Debug Symbols][15].
```

As this page is specific to web apps, we should use the appropriate wording:

Variant | File name | Action verb
-- | -- | --
Generic | Debug symbols | Obfuscate, Deobfuscate
iOS | dSYM | Symbolicate, Desymbolicate
Android | Mapping files | –
JS, React Native | Source maps | –
Flutter | Symbol files | –